docs/_includes/calendar.html

```diff
@@ -157,36 +157,36 @@ <h3>Calendar with selected range</h3>
                     <tr>
                         <td><span>15</span></td>
                         <td class="calendar__range calendar__range--start" aria-selected="true">
```

Comment:
`aria-selected` isn't relevant here on a `td` like this. Plus we have the offscreen text to convey visuals.